packages/cli/generation/ir-migrations/src/migrations/v30-to-v29/migrateFromV30ToV29.ts

```diff
@@ -0,0 +1,357 @@
+import { assertNever } from "@fern-api/core-utils";
+import { GeneratorName } from "@fern-api/generators-configuration";
+import { IrSerialization } from "../../ir-serialization";
```

Comment:
@amckinney you're going to want to update the exports from `IrSerialization`

Reply:
Great catch, updated 👍